### ISSUE TITLE
Interferes with firefox loading webpages

### ISSUE THREAD

**Issue by JJones780** [netblue30/firejail#3773]
**Bug and expected behavior**
Unable to load www.kickstarter.com webpage completely. Doesn't complete loading and unable to interact with horizontal scrolling components. 

**No profile and disabling firejail**
- What changed calling `firejail --noprofile /path/to/program` in a terminal?
Completes loading and I can interact with horizontal scrolling components.
- What changed calling the program by path (check `which <program>` or `firejail --list` while the sandbox is running)?
No difference

**Reproduce**
Linux shell: firejail firefox
Then in the browser enter kickstarter.com into the address bar
Wait for a short while - it may never finish loading
Scroll down to see a row of images with a horizontal scroll bar.
Attempt to grab the scrollbar and scroll horizontally.

**Environment**
 Devuan GNU/Linux 3 (beowulf)
 firejail version 0.9.58.2
 

Note, I also tried the latest stable and nightly versions of firefox ( without any addons, plugins etc. )

**Comment by rusty-snake** [comment]
In my test it worked. Anyway, if it works with `--noprofile`, it can be fixes be improving the profile. Can you try to find out which line in firefox.profile/firefox-common.profile is causing this. (Comment everything and then uncomment line for line).

- Do you have any globals.loca, firefox.local, firefox-common.local/disable-*.local/... or edited firefox.profile/firefox-common.profile/disable-*.inc/...?
- Is anything in the syslog?
- Can you update to a newer firejail version?

> What changed calling the program by path (check which <program> or firejail --list while the sandbox is running)?
No difference

No difference to `--no-profile` or regular? If it happens here, it's not a firejail issue.

**Comment by JJones780** [comment]
I installed firejail 0.9.64 and it solved the issue.  It should've been the first thing I tried ( and I should know better.... )
As a further experiment I tried using the firefox-common.profile from my troubled 0.9.58.2-2+deb10u1 and there was no issue with this old profile ether.

interestingly.. minor recent changes to the kickstarter website had pushed the issue to a secondary page.

Ignore the rest of this -----------------------------------------------------------------------------------------------------  ;)

No customized .local, .profile etc
Nothing in the sysog.  Firefox does crash on closing ( see below) but is unable to write/report a crash log.
Running with profile ( seccomp line) has issue. Running with noprofile or without firejail doesn't exhibit the issue.


Oddly enough I stumbled on another way to replicate the issue after my initial method no longer worked. What are the odds kickstarter changed their webpage between my report and your test the next day?  I better test with the new version of firejail.   I happened to clicked on the '100' banner along the top of the Kickstarter page and that caused the issue reliably.  I was able to investigate further.

firefox-common.profile:   Everything works if I comment out the following (seccomp) line:  

`#seccomp.drop @clock,@cpu-emulation,@debug,@module,@obsolete,@raw-io,@reboot,@resources,@swap,acct,add_key,bpf,fanotify_init,io_cancel,io_destroy,io_getevents,io_setup,io_submit,ioprio_set,kcmp,keyctl,mount,name_to_handle_at,nfsservctl,ni_syscall,open_by_handle_at,personality,pivot_root,process_vm_readv,ptrace,remap_file_pages,request_key,setdomainname,sethostname,syslog,umount,umount2,userfaultfd,vhangup,vmsplice`


The crash on closing firefox ( even when just running with the seccomp line alone enabled ) but is unable to produce a crash report in .mozilla/firefox/Crash.

on shutdown it writes this:
```
[Parent 9, Main Thread] ###!!! ABORT: file /build/firefox-esr-mod3SX/firefox-esr-78.5.0esr/dom/serviceworkers/ServiceWorkerRegistrar.cpp, line 1113
[Parent 9, Main Thread] ###!!! ABORT: file /build/firefox-esr-mod3SX/firefox-esr-78.5.0esr/dom/serviceworkers/ServiceWorkerRegistrar.cpp, line 1113
ExceptionHandler::GenerateDump cloned child 300
ExceptionHandler::WaitForContinueSignal waiting for continue signal...
ExceptionHandler::SendContinueSignalToChild sent continue signal to child
```
